src/main/resources/templates/template_hmpps-approved-premises-api.mustache

```diff
@@ -0,0 +1,755 @@
+                        <tr><td>Historical risk detail</td><td>{{ optionalValue historicalRiskDetail }}</td></tr>
+                        <tr><td>Confirmation</td><td>{{ optionalValue confirmation }}</td></tr>
+                    </table>
+                {{/historical-risk}}
+                {{^historical-risk}}
+                    <p>No Data Held</p>
+                {{/historical-risk}}
+
+                <p><b>Additional information</b></p>
+                {{#additional-information}}
+                    <table class="summary-list">
+                        <tr><td>Has additional information</td><td>{{ optionalValue hasAdditionalInformation }}</td></tr>
+                        <tr><td>Additional information detail</td><td>{{ optionalValue additionalInformationDetail }}</td></tr>
+                    </table>
+                {{/additional-information}}
+                {{^additional-information}}
+                    <p>No Data Held</p>
+                {{/additional-information}}
+
+                <p><b>Vulnerability</b></p>
+                {{#vulnerability}}
+                    <table class="summary-list">
+                        <tr><td>Vulnerability detail</td><td>{{ optionalValue vulnerabilityDetail }}</td></tr>
+                        <tr><td>Confirmation</td><td>{{ optionalValue confirmation }}</td></tr>
+                    </table>
+                {{/vulnerability}}
+                {{^vulnerability}}
+                    <p>No Data Held</p>
+                {{/vulnerability}}
+
+                <p><b>Current risk</b></p>
+                {{#current-risk}}
+                    <table class="summary-list">
+                        <tr><td>Current risk detail</td><td>{{ optionalValue currentRiskDetail }}</td></tr>
+                        <tr><td>Confirmation</td><td>{{ optionalValue confirmation }}</td></tr>
+                    </table>
+                {{/current-risk}}
+                {{^current-risk}}
+                    <p>No Data Held</p>
+                {{/current-risk}}
+
+            {{/data.risk-to-self}}
+            {{^data.risk-to-self}}
+                <p>No Data Held</p>
+            {{/data.risk-to-self}}
+
+            <h5>Community Probation Practitioner details and Home Detention Curfew licence conditions</h5>
+            {{#data.cpp-details-and-hdc-licence-conditions}}
+                <p><b>Community Probation Practitioner details</b></p>
+                {{#cpp-details}}
+                    <table class="data-table">
+                        <tr>
+                            <td class="data-column-25">Name</td>
+                            <td class="data-column-25">Probation region</td>
+                            <td class="data-column-25">Telephone</td>
+                            <td class="data-column-25">Email</td>
+                        </tr>
+                        <tr>
+                            <td>{{ optionalValue (formatDate name) }}</td>
```

Comment:
are these correct with formatDate?

Reply:
good spot, blaming my bad copy and paste skills 😄

Reply:
Removed the unnecessary formatDate calls